Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
         docker {
             image "openshift-art/art-ci-toolkit:latest"
             alwaysPull true
-            args "-e http_proxy -e https_proxy -e no_proxy -e HTTP_PROXY -e HTTPS_PROXY -e NO_PROXY --entrypoint=''"
+            args "-v /home/jenkins/.ssh:/home/art/.ssh -e http_proxy -e https_proxy -e no_proxy -e HTTP_PROXY -e HTTPS_PROXY -e NO_PROXY --entrypoint=''"
```

Comment:
Could you explain why `.ssh` directory is required?

Reply:
I initially didn't manage to have Jenkins fetch tags from remote (tags are required by the new versioning scheme). Setting "Advanced clone behaviours - Fetch tags" flag in [job configuration](https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/art-tools/configure) did the job. Manual fetching can be removed, ssh keys bind mount is no longer necessary